x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/investigation_guide.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiEmptyPrompt, EuiButton, EuiSpacer, EuiMarkdownFormat } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { RuleFormStepId } from '@kbn/response-ops-rule-form/src/constants';
+import React, { useState } from 'react';
+import {
+  AlertDetailsRuleFormFlyout,
+  type AlertDetailsRuleFormFlyoutBaseProps,
+} from './AlertDetailsRuleFormFlyout';
+
+interface InvestigationGuideProps extends AlertDetailsRuleFormFlyoutBaseProps {
+  blob?: string;
+}
+
+export function InvestigationGuide({ blob, onUpdate, refetch, rule }: InvestigationGuideProps) {
+  const [alertDetailsRuleFormFlyoutOpen, setAlertDetailsRuleFormFlyoutOpen] = useState(false);
+  return blob ? (
+    <>
+      <EuiSpacer size="m" />
+      <EuiMarkdownFormat
+        css={css`
+          word-wrap: break-word;
+        `}
+      >
+        {blob}
+      </EuiMarkdownFormat>
+    </>
+  ) : (
+    <>
+      <EuiEmptyPrompt
+        iconType="logoObservability"
+        iconColor="default"
+        title={
+          <h3>
+            <FormattedMessage
+              id="xpack.observability.alertDetails.investigationGide.emptyPrompt.title"
+              defaultMessage="Add an Investigation Guide"
+            />
+          </h3>
+        }
+        titleSize="m"
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.observability.alertDetails.investigationGide.emptyPrompt.body"
+              defaultMessage="Add a guide to your alert's rule."
```

Comment:
Tweaked this to show the value of this feature:

```suggestion
              defaultMessage="Save time by creating an investigation guide that shares step-by-step guidelines or links to helpful resources for investigating alerts."
```

Let me know if the tone is too casual or if the focus is narrow/broad, and I can revise.

Reply:
I am fine with it as well, so we'll go ahead and commit it.